api/api_full.go

```diff
@@ -864,9 +865,11 @@ type FullNode interface {
 	// IPC-specific methods //
 
 	// IPCAddSubnetActor deploys a new subnet actor.
-	IpcAddSubnetActor(ctx context.Context, wallet address.Address, params subnetactor.ConstructParams) (address.Address, error) //perm:write
-	IpcReadGatewayState(ctx context.Context, actor address.Address, tsk types.TipSetKey) (*gateway.State, error)                //perm:read
-	IpcReadSubnetActorState(ctx context.Context, actor address.Address, tsk types.TipSetKey) (*subnetactor.State, error)        //perm:read
+	IpcAddSubnetActor(ctx context.Context, wallet address.Address, params subnetactor.ConstructParams) (address.Address, error)   //perm:write
+	IpcReadGatewayState(ctx context.Context, actor address.Address, tsk types.TipSetKey) (*gateway.State, error)                  //perm:read
+	IpcReadSubnetActorState(ctx context.Context, actor address.Address, tsk types.TipSetKey) (*subnetactor.State, error)          //perm:read
```

Comment:
`k` instead of `tsk`, `gw` instaed of `gatewayAddr` and `p` instead of `params`? The types (subnetactor.ConstructParams and types.TipSetKey ) are descriptive.

Reply:
If you don't mind I'll leave these ones as-is for now to move faster, it would require renamings all over the place to be consistent.